<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="16" total="17" passed="0" failed="0" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-11-10T13:01:34Z" name="Suite Movilidad Socias Web Carmen García" finished-at="2022-11-10T13:01:34Z" duration-ms="10">
    <groups>
    </groups>
    <test started-at="2022-11-10T13:01:34Z" name="Pruebas" finished-at="2022-11-10T13:01:34Z" duration-ms="10">
      <class name="tests.CasosDePrueba">
        <test-method is-config="true" signature="preparacion(org.testng.ITestContext)[pri:0, instance:tests.CasosDePrueba@2118cddf]" started-at="2022-11-10T13:01:34Z" name="preparacion" finished-at="2022-11-10T13:01:34Z" duration-ms="11" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@53976f5c]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
The type java.util.function.Function cannot be resolved. It is indirectly referenced from required .class files
The method until(Function<? super WebDriver,V>) from the type FluentWait<WebDriver> refers to the missing type Function
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
The type java.util.function.Function cannot be resolved. It is indirectly referenced from required .class files
The method until(Function<? super WebDriver,V>) from the type FluentWait<WebDriver> refers to the missing type Function

at utils.ClaseBase.<init>(ClaseBase.java:172)
at pages.HomePage.<init>(HomePage.java:37)
at tests.CasosDePrueba.preparacion(CasosDePrueba.java:42)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preparacion -->
        <test-method is-config="true" signature="preparacion2(org.testng.ITestContext)[pri:0, instance:tests.CasosDePrueba@2118cddf]" started-at="2022-11-10T13:01:34Z" name="preparacion2" finished-at="2022-11-10T13:01:34Z" duration-ms="0" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preparacion2 -->
        <test-method is-config="true" signature="preparacionTests()[pri:0, instance:tests.CasosDePrueba@2118cddf]" started-at="2022-11-10T13:01:34Z" name="preparacionTests" finished-at="2022-11-10T13:01:34Z" duration-ms="0" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preparacionTests -->
        <test-method signature="CP005_IrAPagarMiCredito()[pri:0, instance:tests.CasosDePrueba@2118cddf]" started-at="2022-11-10T13:01:34Z" name="CP005_IrAPagarMiCredito" finished-at="2022-11-10T13:01:34Z" duration-ms="0" status="SKIP">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
The type java.util.function.Function cannot be resolved. It is indirectly referenced from required .class files
The method until(Function<? super WebDriver,V>) from the type FluentWait<WebDriver> refers to the missing type Function
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
The type java.util.function.Function cannot be resolved. It is indirectly referenced from required .class files
The method until(Function<? super WebDriver,V>) from the type FluentWait<WebDriver> refers to the missing type Function

at utils.ClaseBase.<init>(ClaseBase.java:172)
at pages.HomePage.<init>(HomePage.java:37)
at tests.CasosDePrueba.preparacion(CasosDePrueba.java:42)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CP005_IrAPagarMiCredito -->
        <test-method is-config="true" signature="posTests()[pri:0, instance:tests.CasosDePrueba@2118cddf]" started-at="2022-11-10T13:01:34Z" name="posTests" finished-at="2022-11-10T13:01:34Z" duration-ms="0" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- posTests -->
      </class> <!-- tests.CasosDePrueba -->
    </test> <!-- Pruebas -->
  </suite> <!-- Suite Movilidad Socias Web Carmen García -->
</testng-results>
